package.json

```diff
@@ -19,14 +19,17 @@
   },
   "dependencies": {
     "@hapi/joi": "^17.1.1",
+    "@types/busboy": "^0.2.3",
     "avsc": "^5.4.19",
     "axios": "^0.19.2",
     "body-parser": "^1.18.2",
+    "busboy": "^0.3.1",
     "ch-node-session-handler": "git+ssh://git@github.com/companieshouse/node-session-handler.git#c85dad518b56374400ee48ce5340d7fbe24a4a5f",
     "cookie-parser": "^1.4.5",
     "dotenv": "^8.2.0",
     "express": "^4.15.5",
     "express-async-handler": "^1.1.4",
+    "form-data": "^3.0.0",
```

Comment:
Why do we need that? Cannot we just use JSON literals or https://developer.mozilla.org/en-US/docs/Web/API/FormData?

It feels a bit like overkill.